### PR TITLE
931298 - Update the missing keys in the theme UG

### PR DESCRIPTION
**Pull request by Bharathi4332**
Updated the missing themes key in the keys.md file

- SfPdfViewerControlBackgroundColor 
- SfPdfViewerLoadingIndicatorColor

### CODE REVIEW

**Review by Deepak1799 — APPROVED** [review]

File: MAUI/Themes/Keys.md [hunk 1]
```diff
@@ -10629,6 +10629,30 @@ This page lists the keys for each control and the element to which it is mapped
             <br/>
         </td>
     </tr>   
+     <tr>
+        <td>
+          SfPdfViewerControlBackgroundColor
```
Comment (Deepak1799) [inline MAUI/Themes/Keys.md:10634]: Do we mention as "SfPdfViewer" or "SfPdfViewer control" in all other places?

File: MAUI/Themes/Keys.md [hunk 2]
```diff
@@ -10629,6 +10629,30 @@ This page lists the keys for each control and the element to which it is mapped
...
+            <br/>
+            <br/>
+        </td> 
+        <td>
+           Background color of the SfPdfViewer control.
+            <br/>
```
Comment (Deepak1799) [inline MAUI/Themes/Keys.md:10640]: Did we take the description from source or created on own?

image

**Review by Deepak1799 — CHANGES_REQUESTED** [review]

File: MAUI/Themes/Keys.md (hunk 1 above)
Comment (Bharathi4332) [reply MAUI/Themes/Keys.md:10634]: We have mentioned "SfPdfViewer" sir

File: MAUI/Themes/Keys.md (hunk 2 above)
Comment (Bharathi4332) [reply MAUI/Themes/Keys.md:10640]: description is created on my own sir

Comment (Deepak1799) [reply MAUI/Themes/Keys.md:10640]: Cross check whether this key is for document loading or page loading busy indicator
image

**Review by Deepak1799 — CHANGES_REQUESTED** [review]

**Review by Deepak1799 — APPROVED** [review]

File: MAUI/Themes/Keys.md (hunk 2 above)
Comment (Bharathi4332) [reply MAUI/Themes/Keys.md:10640]: It is for both the document and page loading busy indicator

**Pull request merged**